### DISCUSSION TITLE
Is it normal to stockfish not move when playing with the blacks?

### DISCUSSION THREAD

**Discussion by RockatanskyV8** [official-stockfish/Stockfish#3802] [General]
I'm a total beginner in chess as a whole, I'm trying to learn now using Stockfish and Knights as a frontend.

But Stockfish seems to not be able to play with blacks, whenever I start with the whites and make a move, stockfish simply does not make any decision. I see in the terminal here that it is working, but I left it alone for almost an hour and it didn't move nothing.

Is this supose to be this way? Or am I missing something in configurations?

Stockfish 14.1
knights 21.08.3-1

**Comment by vondele** [comment]
what you describe is an issue/property of the Knights frontend, not stockfish. Maybe it defaults to doing an infinite analysis, not playing with time control?

**Comment by RockatanskyV8** [comment]
I tried with time limit, no changes. Still takes a lot of time and no decision from Stockfish
